x-pack/legacy/plugins/siem/public/pages/detection_engine/rules/components/description_step/helpers.tsx

```diff
@@ -266,3 +268,27 @@ export const buildNoteDescription = (label: string, note: string): ListItems[] =
   }
   return [];
 };
+
+export const buildRuleTypeDescription = (label: string, ruleType: RuleType): ListItems[] => {
```

Comment:
@MadameSheema here's the terminal point of those Rule Details changes if you're interested. I haven't addressed the sliders discrepancy but I'll ping you on that when it's resolved!